docs/components/receive.md

```diff
@@ -77,6 +77,23 @@ The example content of `hashring.json`:
 
 With such configuration any receive listens for remote write on `<ip>10908/api/v1/receive` and will forward to correct one in hashring if needed for tenancy and replication.
 
+## Limiting
```

Comment:
Shall we mention this is an experimental feature? I assume since we're hiding the flags for now.

Reply:
Sure!